fiat-web/src/main/java/com/netflix/spinnaker/fiat/controllers/AuthorizeController.java

```diff
@@ -189,6 +200,64 @@ public void getUserAuthorization(
     response.setStatus(HttpServletResponse.SC_NOT_FOUND);
   }
 
+  @RequestMapping(value = "/{userId:.+}/{authorization:.+}", method = RequestMethod.POST)
+  public void hasAuthorization(
+      @PathVariable String userId,
+      @PathVariable String authorization,
+      @RequestBody @Nonnull Resource resource,
```

Comment:
Follow-on to the previous comment - we will need to see how we go from user domain object to Resource but this might need `@PathVariable ResourceType` and `@RequestBody Map<String, Object> resource` so we can handle the object conversion properly - currently the Resource hierarchy isn't set up for Jackson polymorphic deserialization.